src/Sarif.Viewer.VisualStudio/Models/SarifErrorListItem.cs

```diff
@@ -617,32 +599,35 @@ internal void RemapFilePath(string originalPath, string remappedPath)
                     }
                 }
             }
+
+            // After the file-paths have been remapped, we need to refresh the tags
+            // as the persistent spans may now be possible to create (since the file paths are now potential valid)
```

Comment:
>the persistent spans may now be possible to create [](start = 18, length = 50)

"as it may now be possible to create the persistent spans"

Reply:
Sorry :-) "as as" => "as"

---
In reply to: [502561437](https://github.com/microsoft/sarif-visualstudio-extension/pull/214#discussion_r502561437) [](ancestors = 502561437)